### DISCUSSION TITLE
Why the style of PowerShell that "run as administrator" is still like cmd?

### DISCUSSION THREAD

**Discussion by HumanMus1c** [PowerShell/PowerShell#21183] [General]
The left one has set "run this profile as administrator" in settings, and boot directly from start menu.
The another one was booted by right click menu "Run as Administrator".
Why they are different?

![image](https://github.com/PowerShell/PowerShell/assets/46023154/448a6a45-d67e-4aad-a93f-f6dd276651ec)

**Comment by 237dmitry** [comment]
By default there is title. You can modify your custom prompt function to check the current user role.

![Screenshot 2024-02-04 163615](https://github.com/PowerShell/PowerShell/assets/78153320/78badc8e-d068-444d-a54a-5b8ce331b970)

On Linux you can check `id`.

**Reply by HumanMus1c** [reply]
Both default setting were same, but something is different after boot up.

**Reply by HumanMus1c** [reply]
PowerShell 7.4.1 and Windows PowerShell 1.0 both old style by running as Administrator from right contex menu.
I'm just curious why it's not morden layout which could open multiple tabs in one windows?
![image](https://github.com/PowerShell/PowerShell/assets/46023154/1f6d376e-15ab-4452-8390-0dcc80323890)

**Reply by 237dmitry** [reply]
> I'm just curious why it's not morden layout which could open multiple tabs in one windows?

Because you have to set Windows Terminal as default console application. It can be set in "Properties" of console on you screenshot or in "Settings" of Window Terminal or in Windows Settings (don't remember where).

**Reply by HumanMus1c** [reply]
Maybe this is more Intuitive and clear for what was happened exactly.
And I did set "Window Terminal" as default "Properties".
I'm confused as to why these two boot are different.
It's possible to make "run as Administrator" UI same as normal boot one?
Why "run as Administrator" still looks like an old style win32 program?

![image](https://github.com/PowerShell/PowerShell/assets/46023154/849ade56-029b-4e0c-ba31-02e9c5470d54)

![image](https://github.com/PowerShell/PowerShell/assets/46023154/15c6a935-c144-4d13-8d52-5db43060147a)

![recording](https://github.com/PowerShell/PowerShell/assets/46023154/6d92232e-e56c-49ef-8c68-c26ff23c7ee8)
